source/manual/manage-assets.html.md.erb

```diff
@@ -21,24 +21,28 @@ follow these steps:
      The square brackets are part of the command, so don't remove them:
      <%= RunRakeTask.links("asset-manager", "assets:delete[ASSET_ID]") %>
 
-1. Add a query string to the URL (e.g. `?cache-bust=12345`) to bypass the cache
+     For "Whitehall" Assets (that start with the path `/government/uploads/system`), there is a [slightly different Rake task]((https://github.com/alphagov/asset-manager/blob/main/lib/tasks/assets.rake#L9-L13)):
+
+     <%= RunRakeTask.links("asset-manager", "assets:whitehall_delete[ASSET_PATH]") %>
+
+2. Add a query string to the URL (e.g. `?cache-bust=12345`) to bypass the cache
```

Comment:
I suspect that you can leave these '1's as is. They display properly online and this is mirrored in other files;
https://github.com/alphagov/govuk-developer-docs/blob/main/source/manual/pingdom.html.md

Reply:
It pains my OCD to revert this, but it is done. >.<